.vscode/settings.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
We need to add .vscode and .idea to .gitignore.

Reply:
good call